todo-backend-java/src/main/java/todo/backend/FeatureHubSource.java

```diff
@@ -39,20 +38,16 @@ public FeatureHubSource() {
         new GoogleAnalyticsJerseyApiClient()));
     }
 
-    edgeService = new JerseyClient(config, repository);
+    config.setRepository(repository);
+    config.init();
   }
 
   public ClientContext newContext() {
```

Comment:
do you need to rename this to ClientContext fhClient() {} ?

Reply:
forgot about that